source/ed/data.cpp

```diff
@@ -205,9 +204,11 @@ void Data::shift_stop_times() {
         // For non-frequency vj, we must have the first stop time in
         // [0; 24:00[ (since they are ordered, every stop time will be
         // greatter than 0)
+        // Depending on boarding and alighting duration the start_time will be either the arrival_time or
+        // the boarding time.
         //
         // For frequency vj, the start time must be in [0; 24:00[.
-        const int start_time = vj->is_frequency() ? vj->start_time : first_st->arrival_time;
+        const int start_time = vj->is_frequency() ? vj->start_time : std::min(first_st->boarding_time, first_st->arrival_time);
```

Comment:
It's screen resolution dependent no? The line seems ok on a full HD monitor (not so much on a WXGA one though =p). But that's not really important, I'm going to set the limit at 113 on my IDE and change all this :).